backend/src/components/permissionUtils.js

```diff
@@ -154,6 +154,31 @@ function checkSdcSchoolCollectionAccess(req, res, next) {
   }
 }
 
+function loadInstituteCollection(req, res, next) {
```

Comment:
Can we switch this to use res.locals.requestedSdcSchoolCollectionID and res.locals.requestedSdcDistrictCollectionID to determine which sub-load function to use? This will make the dependency on those values within the sub-load a little more easier to see.

Reply:
I dont have sdcSchoolCollectionID and sdcDistrictCollectionID in path params, only in body where both the values will be present, hence using activeInstituteType to differentiate